cookbook/integrations/flytekit_plugins/mlflow_example/mlflow_example.py

```diff
@@ -0,0 +1,70 @@
+"""
+.. _mlflow:
+
+MLflow
+-----
+
+MLflow is a platform to streamline machine learning development, including tracking experiments, packaging code into reproducible runs, and sharing and deploying models.
+
+Flyte provides an easy-to-use interface to log the task's metrics and parameters to either Flyte Deck or MLflow server.
+
+First, install the Flyte MLflow plugin:
+
+.. prompt:: bash $
+
+    pip install flytekitplugins-mlflow
+
+"""
```

Comment:
let's move the installation instructions to the top of `mlflow_example/README.rst` file